src/appmixer/google/gmail/SendEmail/component.json

```diff
@@ -7,7 +7,8 @@
         "service": "appmixer:google",
         "scope": [
             "https://www.googleapis.com/auth/gmail.compose",
-            "https://www.googleapis.com/auth/gmail.send"
+            "https://www.googleapis.com/auth/gmail.send",
+            "https://www.googleapis.com/auth/gmail.modify"
```

Comment:
@vtalas By default gmail.send does not support attaching labels to an email. The labels can only be attached after the email is sent by doing gmail.modify. And for that gmail.modify scope is needed. I have added this scope here. If the logic seems right, we can go ahead with this, otherwise we can remove the label attaching part from the activity